content/cpp/concepts/math-functions/terms/sqrt/sqrt.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: 'sqrt()'
+Description: 'Returns square root of the argument.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`sqrt()`** function returns the square root of the argument.
+
+## Syntax
+
+```cpp
+sqrt(x)
+```
+
+If the parameter `x` is negative (less than 0), then a domain error will occur.
+
+## Example
+
+use `sqrt()` to find the square root of `4`:
```

Comment:
```suggestion
The following example uses the `sqrt()` function to find the square root of `4`:
```

Reply:
The old text is more instructional in tone; Docs entries should be more informational/declarative.